### PR TITLE
librespot: initial addon

### PR DESCRIPTION
**Pull request by awiouy**

### CODE REVIEW

**Comment by awiouy** [comment]
The source of librespot and of the build system used to build the binaries is here:
https://github.com/awiouy/librespot/tree/libreelec

Corresponding upstream pull request is here:
https://github.com/plietar/librespot/pull/186

Binaries are built with:
```
docker build -t librespot-cross -f contrib/Dockerfile .
docker run -v /tmp/librespot-build:/build librespot-cross /src/contrib/docker-build-libreelec.sh
```
Should run on all LibreELEC projects, except RPi

File: packages/addons/service/librespot/changelog.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+100:
```
Comment (CvH) [inline packages/addons/service/librespot/changelog.txt]: @awiouy still not sure about the `:` as we nowhere use it

**Comment by awiouy** [comment]
@cvh: fixed, in LE8, too

**Comment by awiouy** [comment]
Rebased to update version and disable cache

**Comment by awiouy** [comment]
Updated and rebased.

**Comment by awiouy** [comment]
Rebased

**Pull request merged**